core/src/main/java/com/datastax/oss/driver/api/core/cql/Statement.java

```diff
@@ -66,54 +68,72 @@
    * <p>All the driver's built-in implementations are immutable, and return a new instance from this
    * method. However custom implementations may choose to be mutable and return the same instance.
    */
-  T setConfigProfileName(String newConfigProfileName);
+  @NonNull
+  T setConfigProfileName(@NonNull String newConfigProfileName);
```

Comment:
`newConfigProfileName` is nullable, it could be used to remove an existing name. Same for `setConfigProfile`.